<launch>
    <arg name="team" default="unknown"/>
    <arg name="raspberry" default="true" />

    <rosparam command="load" file="$(find strategy)/param/pins.yaml" />
    <rosparam command="load" file="$(find strategy)/param/initial_minus.yaml" />
    <rosparam command="load" file="$(find strategy)/param/game.yaml" />

    <param name="team" value="$(arg team)" />

    <!-- Load the node with different args if the raspberry arg is set -->
    <node if="$(arg raspberry)" name="startup_conf" pkg="strategy" type="startup_conf_minus.py" args="-r" />
    <node unless="$(arg raspberry)" name="startup_conf" pkg="strategy" type="startup_conf_minus.py" />
</launch>
